[2024-02-18T08:59:56.512+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_project.Load_user_dim_table manual__2024-02-18T08:55:13.724214+00:00 [queued]>
[2024-02-18T08:59:56.516+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_project.Load_user_dim_table manual__2024-02-18T08:55:13.724214+00:00 [queued]>
[2024-02-18T08:59:56.517+0000] {taskinstance.py:2171} INFO - Starting attempt 2 of 2
[2024-02-18T08:59:56.523+0000] {taskinstance.py:2192} INFO - Executing <Task(LoadDimensionOperator): Load_user_dim_table> on 2024-02-18 08:55:13.724214+00:00
[2024-02-18T08:59:56.527+0000] {standard_task_runner.py:60} INFO - Started process 23370 to run task
[2024-02-18T08:59:56.530+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'final_project', 'Load_user_dim_table', 'manual__2024-02-18T08:55:13.724214+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/final_project.py', '--cfg-path', '/tmp/tmpmj9q62b7']
[2024-02-18T08:59:56.532+0000] {standard_task_runner.py:88} INFO - Job 74: Subtask Load_user_dim_table
[2024-02-18T08:59:56.557+0000] {task_command.py:423} INFO - Running <TaskInstance: final_project.Load_user_dim_table manual__2024-02-18T08:55:13.724214+00:00 [running]> on host 6b28116ad14a
[2024-02-18T08:59:56.593+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='udacity' AIRFLOW_CTX_DAG_ID='final_project' AIRFLOW_CTX_TASK_ID='Load_user_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-18T08:55:13.724214+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-18T08:55:13.724214+00:00'
[2024-02-18T08:59:56.599+0000] {base.py:83} INFO - Using connection ID 'aws_credentials' for task execution.
[2024-02-18T08:59:56.599+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2024-02-18T08:59:56.605+0000] {load_dimension.py:34} INFO - dim.dim_users is being inserted
[2024-02-18T08:59:56.608+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-18T09:00:00.885+0000] {sql.py:432} INFO - Running statement: 
        SELECT distinct userid, firstname, lastname, gender, level
        into dim.dim_users
        FROM staging.staging_events
        WHERE page='NextSong'
    , parameters: None
[2024-02-18T09:00:07.707+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=final_project, task_id=Load_user_dim_table, execution_date=20240218T085513, start_date=20240218T085956, end_date=20240218T090007
[2024-02-18T09:00:07.746+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-18T09:00:07.761+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
